packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -99,6 +100,8 @@ import {
 } from './util/index.js';
 import { VerticalResizer } from './VerticalResizer.js';
 
+const isFirefox = isFireFoxFn();
```

Comment:
Could lead to an hydration error in SSR environments. This returns always false on the server, on some clients it might be true -> Hydration Error